.golangci.yml

```diff
@@ -13,7 +13,6 @@ linters:
   - megacheck
   - gofmt
   - unconvert
-  - nakedret
```

Comment:
I'm not sure whether this is a net positive. What do we gain by using naked returns in this PR, as opposed to returning `c` and `err`? My other comment suggests rework of the block of code that leverages naked returns; maybe the need for naked returns will go away altogether?

Reply:
I'll rework the relevant functions, added this linter back: 5484248e86d111fa39ded923d1611aed85d413df